packages/graph/hash_graph/tests/integration/postgres/mod.rs

```diff
@@ -328,7 +328,23 @@ impl DatabaseApi<'_> {
         target_entity_id: EntityId,
         link_type_id: VersionedUri,
     ) -> Result<(), InsertionError> {
-        let link = Link::new(source_entity_id, target_entity_id, link_type_id);
+        let link = Link::new(source_entity_id, target_entity_id, link_type_id, None);
+        self.store.create_link(&link, self.account_id).await
+    }
+
+    async fn create_ordered_link(
```

Comment:
Does this need to be a dedicated method? Or can `order` be an optional argument

Reply:
I see, let's go with your judgement on what the more consistent approach is

Reply:
These are mostly helper methods for the 'integration tests' we have in the graph API, so the API surface here isn't too important compared to the datastore interface or the rest API

Reply:
Ah makes sense, thanks for explaining :)